week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -6,18 +6,27 @@
 
   NOTE: only the names should be returned, not the means of transport.
 */
+// function journeyPlanner(select, vehicle) {
```

Comment:
It's normally best to remove commented code when you submit your homework to make the file easier to read

Reply:
Yes, Chris, you are right. I usually do it to keep multiple options or solutions. And sometimes I just forget. Thanks for the comment